<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadarMap</class>
 <widget class="QWidget" name="RadarMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>575</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Настройка отображения радиолокационных данных</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QComboBox" name="typecmb"/>
   </item>
   <item row="0" column="4" rowspan="3" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Слой</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0">
       <widget class="QRadioButton" name="layer_8000_9000">
        <property name="text">
         <string>&amp;8000 - 9000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="layer_5000_6000">
        <property name="text">
         <string>&amp;5000 - 6000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="layer_2000_3000">
        <property name="text">
         <string>&amp;2000 - 3000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="layer_0_1000">
        <property name="text">
         <string>&amp;0 - 1000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="12" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>142</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="layer_1000_2000">
        <property name="text">
         <string>&amp;1000 - 2000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="layer_3000_4000">
        <property name="text">
         <string>&amp;3000 - 4000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="layer_6000_7000">
        <property name="text">
         <string>&amp;6000 - 7000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="layer_4000_5000">
        <property name="text">
         <string>&amp;4000 - 5000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="layer_7000_8000">
        <property name="text">
         <string>&amp;7000 - 8000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QRadioButton" name="layer_9000_10000">
        <property name="text">
         <string>&amp;9000 - 10000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="layer_0_0">
        <property name="text">
         <string>0 - 0</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QRadioButton" name="layer_10000_11000">
        <property name="text">
         <string>10000 - 11000</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">layergroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>351</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Дата:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QDateEdit" name="dateEdit">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTreeWidget" name="radartree">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Дата и время</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество радаров</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>452</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QToolButton" name="btnadd">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QToolButton" name="btndel">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QToolButton" name="btnclose">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="layergroup"/>
 </buttongroups>
</ui>
